tests/unit/conftest.py

```diff
@@ -11,6 +11,7 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 #  or implied. See the License for the specific language governing
 #  permissions and limitations under the License.
+from click.testing import CliRunner
```

Comment:
Ah, one thing I noticed is this import isn't really used anywhere, is it ?

Reply:
you're right. It got left over when I moved the tests from `unit/` to `integration/functional/cli`. I'll remove it.